<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapsDialog</class>
 <widget class="QDialog" name="MapsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapsDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QListWidget" name="mapList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeMapButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="dartTab">
      <attribute name="title">
       <string>DART</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_dart_orbits">
           <property name="text">
            <string>Nb orbits :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dart_orbits"/>
         </item>
         <item>
          <widget class="QLabel" name="label_dart_cells">
           <property name="text">
            <string>Nb cells :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dart_cells"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="dartAttributes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vertexTab">
      <attribute name="title">
       <string>VERTEX</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_vertex_orbits">
           <property name="text">
            <string>Nb orbits :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_vertex_orbits"/>
         </item>
         <item>
          <widget class="QLabel" name="label_vertex_cells">
           <property name="text">
            <string>Nb cells :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_vertex_cells"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="vertexAttributes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="edgeTab">
      <attribute name="title">
       <string>EDGE</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_edge_orbits">
           <property name="text">
            <string>Nb orbits :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_edge_orbits"/>
         </item>
         <item>
          <widget class="QLabel" name="label_edge_cells">
           <property name="text">
            <string>Nb cells :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_edge_cells"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="edgeAttributes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="faceTab">
      <attribute name="title">
       <string>FACE</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_face_orbits">
           <property name="text">
            <string>Nb orbits :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_face_orbits"/>
         </item>
         <item>
          <widget class="QLabel" name="label_face_cells">
           <property name="text">
            <string>Nb cells :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_face_cells"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="faceAttributes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="volumeTab">
      <attribute name="title">
       <string>VOLUME</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_volume_orbits">
           <property name="text">
            <string>Nb orbits :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_volume_orbits"/>
         </item>
         <item>
          <widget class="QLabel" name="label_volume_cells">
           <property name="text">
            <string>Nb cells :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_volume_cells"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="volumeAttributes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
